concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-357	357	to spread, to scatter, to winnow	3705	SCATTER
+RonaTas-2011-431-358	358	grasshopper	607	GRASSHOPPER
+RonaTas-2011-431-359	359	to leap, to jump, to escape, to flee	1829	JUMP
+RonaTas-2011-431-360	360	grape, wine grape, Vitis vinifer	920	GRAPE
+RonaTas-2011-431-361	361	mosquito, gnat, Culex pipens	1509	MOSQUITO
+RonaTas-2011-431-362	362	furrier		
+RonaTas-2011-431-363	363	to cease, to stop	1928	CEASE
+RonaTas-2011-431-364	364	to strain, to filter	940	STRAIN
+RonaTas-2011-431-365	365	virgin, pure	1147	PURE
+RonaTas-2011-431-366	366	sorcerer, medicine man	823	SORCERER
+RonaTas-2011-431-367	367	advice; council		
+RonaTas-2011-431-368	368	witness	1144	WITNESS
+RonaTas-2011-431-369	369	to feed, to nourish	739	FEED
+RonaTas-2011-431-370	370	tinder, tinder fungus, Fomes fomentarius		
+RonaTas-2011-431-371	371	bald	1125	BALD
+RonaTas-2011-431-372	372	depot		
+RonaTas-2011-431-373	373	plough field, arable field, stubble field		
+RonaTas-2011-431-374	374	to hold, carry; to last, visszatart	1448	HOLD
+RonaTas-2011-431-375	375	Tatar		
+RonaTas-2011-431-376	376	name of several kinds of plants, e.g. Anemosphoros, Buniasorientalis, Napus, Crambe tatarica; storm wind		
+RonaTas-2011-431-377	377	to wind something round, to twist	430	TWIST
+RonaTas-2011-431-378	378	trough, wash tub, hutch	205	TROUGH
+RonaTas-2011-431-379	379	strap (on a whip, or on a kind of sandal)	2973	STRAP
+RonaTas-2011-431-380	380	a piece of land, field, parcel, patch, a higher place where animals rest and accumulate dung	212	FIELD
+RonaTas-2011-431-381	381	to vegetate, to linger in misery		
+RonaTas-2011-431-382	382	axle	1769	AXLE
+RonaTas-2011-431-383	383	sea	1474	SEA
+RonaTas-2011-431-384	384	to turn, to change the original direction	1588	TURN
+RonaTas-2011-431-385	385	knee	1371	KNEE
+RonaTas-2011-431-386	386	hall, chamber, great room		
+RonaTas-2011-431-387	387	camel	1871	CAMEL
+RonaTas-2011-431-388	388	hemp breaker, swingle, scutch		
+RonaTas-2011-431-389	389	steer, young bullock, ox	1169	OX
+RonaTas-2011-431-390	390	to lay eggs	2464	LAY EGGS
+RonaTas-2011-431-391	391	sturgeon, Acipenser sturio	3606	STURGEON
+RonaTas-2011-431-392	392	one-year-old lamb		
+RonaTas-2011-431-393	393	interpreter	3744	INTERPRETER
+RonaTas-2011-431-394	394	horseradish, Armoracia rusticana		
+RonaTas-2011-431-395	395	to pile up, to become blocked	1576	PILE UP
+RonaTas-2011-431-396	396	a kind of small falcon, Falco columbarius aescalon		
+RonaTas-2011-431-397	397	Slovak		
+RonaTas-2011-431-398	398	perfect		
+RonaTas-2011-431-399	399	innumerable		
+RonaTas-2011-431-400	400	to break (also e.g. hemp, salt, pepper), to separate into pieces, to break,to crush	1520	CRUSH
+RonaTas-2011-431-401	401	snare, trap	1075	TRAP (PITFALL)
+RonaTas-2011-431-402	402	Turk, Turkish, Turkic, Ottoman		
+RonaTas-2011-431-403	403	to happen, to occur	248	HAPPEN
+RonaTas-2011-431-404	404	law	1062	LAW
+RonaTas-2011-431-405	405	peat, turf, dried dung		
+RonaTas-2011-431-406	406	steer, young bullock, ox or cow	1007	COW
```

Comment:
Is this a duplicate to 389? In any case, I'd unmap for the same reasons as above.

Reply:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534